packages/@react-spectrum/picker/intl/it-IT.json

```diff
@@ -1,4 +1,4 @@
 {
   "loading": "Caricamento...",
-  "placeholder": "Seleccione una opción…"
+  "placeholder": "Seleziona un’opzione..."
 }
```

Comment:
This change was intended for es-ES. Reverting it-IT to original translation, and applying intended fix to es-ES